projects/aca-testing-shared/src/pages/page.ts

```diff
@@ -82,13 +83,11 @@ export abstract class Page {
   }
 
   async isUploadDialogOpen(): Promise<boolean> {
-    return isPresentAndDisplayed(this.uploadDialog);
+    return this.uploadFilesDialog.isUploadDialogOpen();
```

Comment:
You can remove this one line method and call it directly in spec file

Reply:
Fixed